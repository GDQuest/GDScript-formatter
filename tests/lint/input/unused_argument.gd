#region Good

func good(used_arg, _unused_arg):
	print(used_arg)


func set_selection(node: Node) -> void:
	something.get_selection().add_node(node)
	something.edit_node(node)

#endregion

#region Bad

func bad(used_arg, unused_arg):
	print(used_arg)

#endregion
