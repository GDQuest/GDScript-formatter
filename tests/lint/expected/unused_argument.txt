tests/lint/input/unused_argument.gd:15:unused-argument:warning: Function argument 'unused_arg' is unused. Consider removing it or prefixing with '_'